lib/recurly/invoice.php

```diff
@@ -69,7 +69,18 @@ public function markSuccessful() {
   public function markFailed() {
     $this->_save(Recurly_Client::PUT, $this->uri() . '/mark_failed');
   }
-
+  
+  /**
+   * Enters an offline payment for an invoice (beta)
+   * @param array additional transaction attributes (see writeableAttributes for Recurly_Transaction)
```

Comment:
I'm wondering if it would be better to just spell out `'account','amount_in_cents','currency','description','accounting_code'`?